pkgs/applications/radio/tlf/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, pkgconfig, glib, perl, ncurses
+, hamlib, xmlrpc_c }:
+
+stdenv.mkDerivation rec {
+  pname = "tlf";
+  version = "1.3.2";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "${pname}-${version}";
+    sha256 = "0gniysjm8aq5anq0a0az31vd6h1vyg56bifc7rpf53lsh9hkzmgc";
+  };
+
+  nativeBuildInputs = [ autoconf automake pkgconfig perl ];
+  buildInputs = [ glib ncurses hamlib xmlrpc_c ];
+
+  preConfigure = "autoreconf --install";
```

Comment:
That can be simplified by just placing `autoreconfHook` into `nativeBuildInputs`.